docs/architecture/grpc-for-wcf-developers/protobuf-maps.md

```diff
@@ -1,11 +1,36 @@
 ---
 title: Protobuf maps for dictionaries - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: Using Protobuf maps to represent .NET's Dictionary types
 author: markrendle
-ms.date: 09/02/2019
+ms.date: 09/09/2019
 ---
 
 # Protobuf maps for dictionaries
 
+Protobuf's equivalent of the .NET `Dictionary<TKey, TValue>` type is `map<key_type, value_type>`.
```

Comment:
This article is missing an intro of what's going to cover. This is super important for SEO.